neutrino-custom/production.js

```diff
@@ -1,48 +1,26 @@
 'use strict';
-const webpack = require('webpack');
-const basePreset = require('./base');
-const CopyPlugin = require('copy-webpack-plugin');
-const CleanPlugin = require('clean-webpack-plugin');
-const CWD = require('./base').CWD;
-const UI = require('./base').UI;
-const DIST = require('./base').DIST;
-
-// The service domain is used to determine whether login is available in the auth component.
-let SERVICE_DOMAIN = process.env.SERVICE_DOMAIN;
+const { DefinePlugin } = require('webpack');
+const copy = require('neutrino-middleware-copy');
 
 module.exports = neutrino => {
-    basePreset(neutrino);
-
-    neutrino.config.plugin('minify')
-        .inject(BabiliPlugin => new BabiliPlugin({
-            evaluate: false, // prevents some minification errors
-        }
-    ));
-
-    // Define the service domain globally so that window.thServiceDomain can be set:
-    neutrino.config.plugin('define')
-        .use(webpack.DefinePlugin, {
-            SERVICE_DOMAIN: JSON.stringify(SERVICE_DOMAIN)
-        });
-
+    neutrino.config
+        .plugin('minify')
+        // prevents some minification errors
+            .tap(() => [{ evaluate: false }]).end()
```

Comment:
Neutrino v5 switches the devtool in development from `eval` to `source-map`. Maybe this isn't needed any more with v5?